pinot-query-planner/src/main/java/org/apache/pinot/calcite/sql/fun/PinotOperatorTable.java

```diff
@@ -46,127 +59,296 @@
  * </ul>
  */
 @SuppressWarnings("unused") // unused fields are accessed by reflection
-public class PinotOperatorTable extends SqlStdOperatorTable {
-
-  private static @MonotonicNonNull PinotOperatorTable _instance;
-
-  // TODO: clean up lazy init by using Suppliers.memorized(this::computeInstance) and make getter wrapped around
-  // supplier instance. this should replace all lazy init static objects in the codebase
-  public static synchronized PinotOperatorTable instance() {
-    if (_instance == null) {
-      // Creates and initializes the standard operator table.
-      // Uses two-phase construction, because we can't initialize the
-      // table until the constructor of the sub-class has completed.
-      _instance = new PinotOperatorTable();
-      _instance.initNoDuplicate();
-    }
-    return _instance;
+public class PinotOperatorTable implements SqlOperatorTable {
```

Comment:
This class is the one that is used at parsing time? ie here we enumerate operator signatures (name, types, etc) but not semantic, right? Can we also add that to the javadoc?

Reply:
Good catch. I forgot to update the javadoc for this class